jnigen/test/simple_package_test/java/com/github/dart_lang/jnigen/generics/MyStack.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Please also add a MyHashMap, so that we have an example with two type arguments and with methods which only use one of the type arguments.

Reply:
That is the plan!